.github/workflows/ci.yaml

```diff
@@ -70,8 +84,28 @@ jobs:
       - name: Install dependencies
         run: poetry install --sync
 
+      - name: Configure ECR AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::258143015559:role/github-amazonec2containerregistryreadonly
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v2
+
+      - name: Start services
+        run: |
+          docker compose -f docker-compose.yml up -d --quiet-pull --wait --wait-timeout 150
```

Comment:
This should be a sequence of commands, see [containerize-agent.yaml](https://github.com/great-expectations/cloud/blob/85bfeebe954b03b4c51503bc2575b324a8767685/.github/workflows/containerize-agent.yaml):
```
          docker compose up db mq -d
          docker compose run db-provisioner
          docker compose run db-seeder
          docker compose run mq-wait
          docker compose up mercury-service-api -d
          docker compose up gx-agent -d
          poetry run pytest -m "agentjobs"
```